supervisory-authorities/derf.json

```diff
@@ -0,0 +1,16 @@
+{
+    "slug": "derf",
+    "relevant-countries": [
+        "de"
+    ],
+    "name": "Der Rundfunkdatenschutzbeauftragte von BR, SR, WDR, Deutschlandradio und ZDF",
+    "address": "Marlene-Dietrich-Allee 20\n14482 Potsdam\nDeutschland",
+    "phone": "+49 331 9799385500",
```

Comment:
The same phone number is also listed in their legal notice: https://www.rundfunkdatenschutz.de/impressum/

Let's switch to that as the source and also add the fax number.